HealthTracker/App.tsx

```diff
@@ -1,29 +1,30 @@
 import { StatusBar } from "expo-status-bar";
 import React, { useState } from "react";
-import {
-  StyleSheet,
-  View,
-  Text,
-  Button,
-  Modal,
-  TouchableOpacity,
-} from "react-native";
+import { View, Text, Modal, TouchableOpacity } from "react-native";
 import { styles } from "./styles";
 import { Feather } from "@expo/vector-icons";
 import { MaterialCommunityIcons } from "@expo/vector-icons";
 import IconButton from "./components/IconButton";
 import BloodPressureForm from "./feature/BloodPressureForm";
 import { BloodPressureData } from "./feature/BloodPressureForm";
+import CameraModal from "./components/CameraModal";
 
 export default function App() {
   const [isModalVisible, setModalVisible] = useState(false);
+  const [isCameraVisible, setCameraVisible] = useState(false);
   const [lastRecord, setLastRecord] = useState<BloodPressureData | null>(null);
+  const [capturedPhoto, setCapturedPhoto] = useState<string | null>(null);
```

Comment:
Capture photo has not been use it, please add it in the feature adn consider a useReducer whit he isCameraVisible useState